src/coreclr/vm/comdelegate.cpp

```diff
@@ -2263,17 +2251,15 @@ extern "C" PCODE QCALLTYPE Delegate_GetMulticastInvokeSlow(MethodTable* pDelegat
                                                                pSig, cbSig,
                                                                NULL,
                                                                &sl);
-        pStub = Stub::NewStub(JitILStub(pStubMD));
+        pStub = JitILStub(pStubMD);
```

Comment:
This is fixing memory leak. Stub allocated by Stub::NewStub would be never freed with collectible assemblies. We do not need to allocate it in the first place.